newton/tests/test_recorder.py

```
@@ -31,6 +34,184 @@ class TestRecorder(unittest.TestCase):
     pass
 
 
+def test_ringbuffer_basic(test: TestRecorder, device):
```

Ruff (0.12.2)

Unused function argument: `device`

(ARG001)